app/policies/comment_policy.rb

```diff
@@ -15,11 +15,11 @@ def create?
   end
 
   def update?
-    true if @current_user.admin? && @current_user.id == @comment.author_id
+    @current_user.admin? && @current_user.id == @comment.author_id
```

Comment:
`@current_user.admin? && @current_user.id == @comment.author_id` I think you have a bug here. Can you tell why?